iogt/views.py

```diff
@@ -121,10 +121,12 @@ def get(self, request, page_id):
                 page_urls.append(page.url)
         translation.activate(active_locale.language_code)
         image_urls = []
+        media_urls = []
         for page in pages:
-            if isinstance(page, (HomePage, Section, Article, Poll, Survey, Quiz)):
-                page_urls.append(page.url)
-                image_urls += page.get_image_urls
+            try:
+                page_urls.append(page.offline_urls)
+            except AttributeError:
+                pass
```

Comment:
Could you add a warning log here to describe the situation and on which page type it's occurring?

Reply:
Added. Sample log, based on data from sandbox:

```
Offline URLs could not be obtained for the following Page types: BannerIndexPage, FooterIndexPage, MiscellaneousIndexPage, PageLinkPage, PollIndexPage, QuizIndexPage, SectionIndexPage, SurveyIndexPage
```